tests/languages/bash/function_feature.test

```diff
@@ -1,101 +1,93 @@
+	["function", "lpc"], ["function", "lpr"], ["function", "lprint"], ["function", 
+	"lprintd"], ["function", "lprintq"], ["function", "lprm"], ["function", "ls"], 
+	["function", "lsof"], ["function", "make"], ["function", "man"], ["function", 
+	"mkdir"], ["function", "mkfifo"], ["function", "mkisofs"], ["function", 
+	"mknod"], ["function", "more"], ["function", "most"], ["function", "mount"], 
+	["function", "mtools"], ["function", "mtr"], ["function", "mv"], ["function", 
+	"mmv"], ["function", "nano"], ["function", "netstat"], ["function", "nice"], 
+	["function", "nl"], ["function", "nohup"], ["function", "notify-send"], 
+	["function", "npm"], ["function", "nslookup"], ["function", "open"], 
+	["function", "op"], ["function", "passwd"], ["function", "paste"], ["function", 
+	"pathchk"], ["function", "ping"], ["function", "pkill"], ["function", "popd"], 
+	["function", "pr"], ["function", "printcap"], ["function", "printenv"], 
+	["function", "ps"], ["function", "pushd"], ["function", "pv"], ["function", 
+	"quota"], ["function", "quotacheck"], ["function", "quotactl"], ["function", 
+	"ram"], ["function", "rar"], ["function", "rcp"], ["function", "reboot"], 
+	["function", "rename"], ["function", "renice"], ["function", "remsync"], 
+	["function", "rev"], ["function", "rm"], ["function", "rmdir"], ["function", 
+	"rsync"], ["function", "screen"], ["function", "scp"], ["function", "sdiff"], 
+	["function", "sed"], ["function", "seq"], ["function", "service"], ["function", 
+	"shellcheck"], ["function", "shuf"], ["function", "sftp"], ["function", "sh"], 
+	["function", "shutdown"], ["function", "sleep"], ["function", "slocate"], 
+	["function", "sort"], ["function", "split"], ["function", "ssh"], ["function", 
+	"stat"], ["function", "strace"], ["function", "su"], ["function", "sudo"], 
+	["function", "sum"], ["function", "suspend"], ["function", "sync"], 
+	["function", "tac"], ["function", "tail"], ["function", "tar"], ["function", 
+	"tee"], ["function", "time"], ["function", "timeout"], ["function", "touch"], 
+	["function", "top"], ["function", "traceroute"], ["function", "tr"], 
+	["function", "tsort"], ["function", "tty"], ["function", "umount"], 
+	["function", "uname"], ["function", "unexpand"], ["function", "uniq"], 
+	["function", "units"], ["function", "unrar"], ["function", "unshar"], 
+	["function", "uptime"], ["function", "useradd"], ["function", "userdel"], 
+	["function", "usermod"], ["function", "users"], ["function", "uuencode"], 
+	["function", "uudecode"], ["function", "v"], ["function", "vdir"], ["function", 
+	"vi"], ["function", "vmstat"], ["function", "wait"], ["function", "watch"], 
+	["function", "wc"], ["function", "wget"], ["function", "whereis"], ["function", 
+	"which"], ["function", "who"], ["function", "whoami"], ["function", "write"], 
+	["function", "xargs"], ["function", "xdg-open"], ["function", "yes"], 
+	["function", "zenity"], ["function", "zip"], ["function", "zsh"]

```

Comment:
Would you mind put each function on its own line? The readability would improve a lot.

```
apropos
apt
apt-get
aptitude
aspell
...
```

```
    ["function", "apropos"],
    ["function", "apt"],
    ["function", "apt-get"],
    ["function", "aptitude"],
    ["function", "aspell"],
    ...
```

Reply:
Also, it would be great to have a similar file `builtin_feature.test` to test the `builtin` pattern you added.